pipit/tests/trace.py

```diff
@@ -185,34 +187,34 @@ def test_time_profile(data_dir, ping_pong_otf2_trace):
 
     # check against ground truth
     # generated using Vampir's Function Summary chart (step size=16)
-    assert np.isclose(norm.loc[0]["int main(int, char**)"], 0.00299437)
-    assert np.isclose(norm.loc[0]["MPI_Init"], 0.93999815)
-    assert np.isclose(norm.loc[0]["MPI_Comm_size"], 0.0)
-    assert np.isclose(norm.loc[0]["MPI_Comm_rank"], 0.0)
-    assert np.isclose(norm.loc[0]["MPI_Send"], 0.0)
-    assert np.isclose(norm.loc[0]["MPI_Recv"], 0.0)
-    assert np.isclose(norm.loc[0]["MPI_Finalize"], 0.0)
-
-    assert np.isclose(norm.loc[1:59]["int main(int, char**)"], 0.0).all()
-    assert np.isclose(norm.loc[1:59]["MPI_Init"], 1.0).all()
-    assert np.isclose(norm.loc[1:59]["MPI_Comm_size"], 0.0).all()
-    assert np.isclose(norm.loc[1:59]["MPI_Comm_rank"], 0.0).all()
-    assert np.isclose(norm.loc[1:59]["MPI_Send"], 0.0).all()
-    assert np.isclose(norm.loc[1:59]["MPI_Recv"], 0.0).all()
-    assert np.isclose(norm.loc[1:59]["MPI_Finalize"], 0.0).all()
-
-    assert np.isclose(norm.loc[60]["int main(int, char**)"], 0.39464799)
-    assert np.isclose(norm.loc[60]["MPI_Init"], 0.14843661)
-    assert np.isclose(norm.loc[60]["MPI_Send"], 0.24594134)
-    assert np.isclose(norm.loc[60]["MPI_Recv"], 0.21017099)
-    assert np.isclose(norm.loc[60]["MPI_Comm_size"], 0.00046047)
-    assert np.isclose(norm.loc[60]["MPI_Comm_rank"], 0.00034261)
-    assert np.isclose(norm.loc[60]["MPI_Finalize"], 0.0)
-
-    assert np.isclose(norm.loc[61]["int main(int, char**)"], 0.43560727)
-    assert np.isclose(norm.loc[61]["MPI_Init"], 0.0)
-    assert np.isclose(norm.loc[61]["MPI_Send"], 0.29640222)
-    assert np.isclose(norm.loc[61]["MPI_Recv"], 0.24300865)
-    assert np.isclose(norm.loc[61]["MPI_Comm_size"], 0.0)
-    assert np.isclose(norm.loc[61]["MPI_Comm_rank"], 0.0)
-    assert np.isclose(norm.loc[61]["MPI_Finalize"], 0.01614835)
+    assert_allclose(norm.loc[0]["int main(int, char**)"], 0.00299437, rtol=1e-05)
```

Comment:
`assert_allclose` is the cleaner/more recommended way for comparing numpy arrays.

I had to adjust the tolerances since assert_allclose uses stricter tolerances than isclose for some of them.
(I'm not sure how the expected numbers were acquired, but I'm guessing that some of them were rounded which would make us off by a little here).